videos/models.py

```diff
@@ -47,3 +47,5 @@ class VideoScore(models.Model):
     video = models.ForeignKey('Video')
     score = models.IntegerField()
 
+    def __str__(self):
+        return 'User:' + str(self.user) + 'Video' + str(self.video)
```

Comment:
Would probably be helpful to show the score in addition to the user and the video. method looks good.